Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Agents"
 uuid = "46ada45e-f475-11e8-01d0-f70cc89e6671"
 authors = ["Tim DuBois", "George Datseris", "Ali Vahdati"]
-version = "4.2.5"
+version = "4.2.6"
```

Comment:
Please see https://semver.org/ . You have to increment the MINOR version

Reply:
Didn't know that, thanks. I'll keep it in mind

Reply:
This would become 4.3.0, right?